config/webpack.common.js

```diff
@@ -126,23 +130,26 @@ module.exports = (env) => {
     },
     output: {
       filename: '[name].bundle.js',
-      path: path.resolve(__dirname, 'dist'),
+      path: path.resolve(__dirname, '../dist'),
     },
     plugins: [
       new HtmlWebpackPlugin(
         env === 'development' || process.env.DATA_SOURCE === 'mock'
           ? {
               // In dev and mock-prod modes, populate window._env at build time
               filename: 'index.html',
```

Comment:
To your earlier point, maybe we can remove this line since index.html is the default. Doesn't really matter though.

Reply:
I'll leave it since it provides a bit more clarity as to what webpack is doing.